packages/babel-helpers/src/helpers/set.ts

```diff
@@ -43,10 +53,16 @@ function set(target, property, value, receiver) {
     };
   }
 
-  return set(target, property, value, receiver);
+  return setImpl(target, property, value, receiver);
 }
 
-export default function _set(target, property, value, receiver, isStrict) {
+export default function _set(
+  target: Object,
+  property: string | symbol,
+  value: any,
+  receiver?: any,
+  isStrict: boolean = false,
```

Comment:
```suggestion
  isStrict?: boolean,
```

This file needs to be compatible with old browsers, so let's not use default params :)